lib/3scale_toolbox/commands/remote_command/remote_rename.rb

```diff
@@ -0,0 +1,52 @@
+require 'cri'
+require '3scale_toolbox/base_command'
+
+module ThreeScaleToolbox
+  module Commands
+    module RemoteCommand
+      class RemoteRenameSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+        def self.command
+          Cri::Command.define do
+            name        'rename'
+            usage       'rename <remote_old_name> <remote_new_name>'
+            summary     '3scale CLI remote rename'
+            description '3scale CLI command to rename remote name'
+            param       :remote_old_name
+            param       :remote_new_name
+            runner RemoteRenameSubcommand
+          end
+        end
+
+        def run
+          # 'arguments' cannot be converted to Hash
+          rename_remote arguments[:remote_old_name], arguments[:remote_new_name]
+        rescue StandardError => e
+          warn e.message
+          exit 1
+        end
+
+        def validate_remote_old_name(name)
+          remotes = config.data :remotes
+          raise "Could not rename, old name '#{name}' does not exist." unless !remotes.nil? && remotes.key?(name)
```

Comment:
It is always a bad idea to use `unless ... && ...` for readability reasons. Also this condition is used all over the place and IMO it can be cleaned up. Either by abstracting the whole "remote store" to own object or by some utility functions.